docs/developers/new_dataset_to_armory.md

```diff
@@ -0,0 +1,120 @@
+# How to add a new dataset into armory
+I will use the RAIT dataset as an example. It is an object detection dataset.
+
+## Step 1 download dataset and locate all file in a folder
+
+Use the RAITE dataset as an example. The train/test json files are loaded in which contain the image name, label, bboxes, and etc.
```

Comment:
"and etc." is redundant since "et cetera" means "and others"

Do be explicit here, what exactly do the json files contain? image name is likely obvious. The label could be ordinal, string enumeration. How many bboxes and what label applies to them? What is "and others"?

A sample, possibly abbreviated record example would really help here.

Reply:
/What is the test / train split? How will the subsets be used by armory?